src/stavisky_lab_to_nwb/simulated_data/simulated_datatrialsinterface.py

```diff
@@ -0,0 +1,114 @@
+"""Primary class for converting trial data."""
+import redis
+import numpy as np
+from pathlib import Path
+from typing import Optional
+
+from pynwb.file import NWBFile
+
+from neuroconv.basedatainterface import BaseDataInterface
+
+
+class StaviskyTrialsInterface(BaseDataInterface):
+    """Trials interface for simulated_data conversion"""
+    
+    def __init__(
+        self, 
+        port: int,
+        host: str,
+    ):
+        super().__init__(port=port, host=host)
+
+    def get_metadata(self):
+        # Automatically retrieve as much metadata as possible
+        metadata = super().get_metadata()
+        
+        return metadata
+    
+    def run_conversion(
+        self,
+        nwbfile: NWBFile,
+        metadata: Optional[dict] = None,
+        stub_test: bool = False,
+    ):
+        # Instantiate Redis client and check connection
+        r = redis.Redis(
+            port=self.source_data["port"],
+            host=self.source_data["host"],
+        )
+        r.ping()
+        
+        # Get session start time to reference
+        session_start_time = np.frombuffer(
+            r.xrange('metadata')[0][1][b'startTime'],
+            dtype=np.float64).item()
+        
+        # Extract trial information
+        trial_info = r.xrange("trial_info", count=(10 if stub_test else None))
+        task_state = r.xrange("task_state", count=(10 if stub_test else None))
```

Comment:
Stub test is mainly for bulk data; that is, the main continuous stream for the ecephys data

Shouldn't need it for light-weight trials data